.github/workflows/test.yml

```diff
@@ -19,10 +19,20 @@ jobs:
     - name: Checkout code
       uses: actions/checkout@v4
 
+    - name: Install Microsoft Go
+      run: |
+        GO_VERSION=$(sed -n 3p ./go.mod | awk '{ print $2; }')
+        curl -sSLf -o "$RUNNER_TEMP/msgo.tgz" https://aka.ms/golang/release/latest/go${GO_VERSION}-1.linux-amd64.tar.gz
+        [[ -d "$RUNNER_TEMP/bin" ]] || install -d -m 0755 "$RUNNER_TEMP/bin"
+        [[ -d "$RUNNER_TEMP/microsoft" ]] || install -d -m 0755 "$RUNNER_TEMP/microsoft"
+        tar -C "$RUNNER_TEMP/microsoft" -xf "$RUNNER_TEMP/msgo.tgz"
+        [[ -x "$RUNNER_TEMP/microsoft/go/bin/go" ]] && ln -s "$RUNNER_TEMP/microsoft/go/bin/go" "$RUNNER_TEMP/bin/microsoft-go"
+        echo "$RUNNER_TEMP/bin" >> "$GITHUB_PATH"
+
     - name: Install Go
       uses: actions/setup-go@v5
       with:
-        go-version: oldstable
+        go-version: stable
```

Comment:
Good catch! We should definitely run the tests against the newer version and only test the old version in `cross_build.yml`

Reply:
Realized I missed this in the release.yml so changed oldstable->stable there, too.